pages/applicants.tsx

```diff
@@ -0,0 +1,284 @@
+import type { NextPage } from 'next';
+import styles from '../styles/Applicants.module.scss';
+import Image, { StaticImageData } from 'next/image';
+import { BellIcon, GiftIcon } from '@heroicons/react/24/outline';
+import AdminNavigation from 'components/molecules/Navigation/AdminNavigation';
+
+
+import Button from 'components/atoms/Button/Button';
+import CheckBox from 'components/atoms/CheckBox/CheckBox';
+import DropDown from 'components/atoms/DropDown/DropDown';
+import SearchBar from 'components/atoms/SearchBar/SearchBar';
+
+import ProfileImage from '../assets/applications/ProfilePic.png';
+import FrontDevLady from '../assets/applications/FrontDevLady.png';
+import { useState } from 'react';
+
+
+export type Applicant = {
+    id: number;
+    name: string;
+    imageUrl: StaticImageData;
+    role: string;
+    isSelected: boolean;
+}
+
+
+const Applicants: NextPage = () => {
+
+    const [isOpen, setIsOpen] = useState(false);
```

Comment:
Set types for all React.useState

https://linguinecode.com/post/how-to-use-react-usestate-with-typescript
https://bobbyhadz.com/blog/react-type-usestate-object